wsd-frontend/src/components/wsd/MemeComment/MemeComment.tsx

```diff
@@ -114,6 +167,14 @@ export function MemeComment({ comment }: { comment: Includes<APIType<'PostCommen
             >
               <Icons.ArrowBigDown size={20} className={cn(feedback === -1 && 'text-destructive fill-destructive')} />
             </Button>
+            <Button
+              onClick={handleShare}
+              size="sm"
+              className="flex items-center gap-1 p-2 rounded-md transition-colors text-gray-500 hover:bg-secondary bg-transparent"
+              aria-label="Share"
+            >
+              <Icons.LucideLink size={20} />
```

Comment:
Can you try 16 instead of 20, that looks more natural, somehow the link icon is bigger than others (maybe has some padding baked in to the svg) Also another nit, can you change it to Icons.Link instead of Icons.LucideLink